src/toolbar/toolbar-controller.js

```diff
@@ -34,6 +54,21 @@
 
         normalSizeClick() {
             this.window.restore();
+            this.window.resizeTo(1280, 720, 'top-right');
+        }
+
+        summariseClick() {
+            this.summarised = !this.summarised;
+            this.window.summarised = this.summarised;
+            if (this.summarised) {
+                this.window.resizeTo(230, 500, 'top-right');
+            }
+            else if (this.maximised) {
+                this.window.maximize();
+            }
+            else {
+                this.window.resizeTo(1280, 720, 'top-right');
```

Comment:
It might be nice to store the window size (and maybe even position?) when the window is changed into compact view. Then when the window is restored the old size could also be restored, instead of using the default?

This is more of a feature though perhaps.